app/src/sharedTest/java/org/oppia/app/utility/FontSizeMatcher.kt

```diff
@@ -0,0 +1,43 @@
+package org.oppia.app.utility
+
+import android.view.View
+import android.widget.TextView
+import androidx.annotation.CheckResult
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.TypeSafeMatcher
+
+// https://stackoverflow.com/questions/47398659/check-the-font-size-height-and-width-of-the-edittext-with-espresso
+/** This class mainly provides a custom matcher to test whether the text matches expected font size. */
+class FontSizeMatcher(expectedSize: Float) :
```

Comment:
Nit update: `class FontSizeMatcher(private val expectedSize: Float) :` and after this you can remove the `private val expectedSize: Float`

Reply:
Done.